template.distillery/README.md

```diff
@@ -148,8 +148,19 @@ of Android packages ...
 - Install JDK 7 or newer (`openjdk-7-jdk` package in Debian/Ubuntu)
 - Install Gradle (`gradle` package in Debian/Ubuntu)
 - Download and untar the [Android SDK](http://developer.android.com) (the smaller version without Android Studio suffices), rename it so that you have a `$HOME/android-sdk-linux/tools` folder.
-- Using the Android package management interface (or `sdkmanager`), install latest versions of SDK Tools (`tools`), SDK Platform-tools (`platform-tools`), and SDK Build-tools (`build-tools;XXX`)
-- Install recent Android API: SDK Platform (`platforms;XXX`).
+- Using the Android package management interface (or `sdkmanager`), install latest versions of SDK Tools (`tools`), SDK Platform-tools (`platform-tools`), and SDK Build-tools (`build-tools;XXX`). From your `android-sdk-linux/tools` :
+```Shell
```

Comment:
Add a comment to tell user to adapt version number (we won't change the documentation everytime there is a new version).